lib/vagrant-service-manager/plugin_util.rb

```diff
@@ -162,9 +162,11 @@ def self.fetch_existing_oc_binary_path_in_windows
         path_string = ENV['PATH']
 
         separator = ':' unless path_string.include?(';')
-        path_string.split(separator).detect do |dir_path|
+        oc_path_dir = path_string.split(separator).detect do |dir_path|
```

Comment:
@hferentschik Yes, I forgot to return the detected oc.exe path. Instead I was returning the first detected `dir_path`